src/XrdCns/XrdCnsLogRec.hh

```diff
@@ -58,8 +58,8 @@ struct Ctl
 
 struct Arg
       {char Type;                   // Event code
-       char Mode[3];                // Mode (create, inv, mkdir)
-       char SorT[12];               // Size (closew, inv) | TOD (eol)
```

Comment:
In these two cases we always lost the last char, because `snprintf` needs to write terminating NULL byte. Thus we need to increase buffers in order not to loose the last digit.